### PR TITLE
Fixed issue #202

### PR DESCRIPTION
**Pull request by nnabuihe-favour**
Appropriate fix for issue #202 without env conflicts. please neglect the previous commit

### CODE REVIEW

File: saskatoon/sitebase/static/css/bootstrap4.min.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*!
+ * Bootstrap v4.0.0 (https://getbootstrap.com)
```
Comment (tristanlatr) [inline saskatoon/sitebase/static/css/bootstrap4.min.css:2]: i think we're already using bootstrap v3 somewhere, I' mo unsure if we should use both...

**Review by tristanlatr — CHANGES_REQUESTED** [review]
Is is possible to make theses changes without included bootstrap4 as a whole ? 
If we're using bootstrap4, I think we should drop bootstrap3, having both could introduce conflicts.

File: saskatoon/sitebase/static/css/bootstrap4.min.css (hunk 1 above)
Comment (nnabuihe-favour) [reply saskatoon/sitebase/static/css/bootstrap4.min.css:2]: yes the bootstrap v4 is just for the crispy form. the rest of the codebase still uses v3

Comment (nnabuihe-favour) [reply saskatoon/sitebase/static/css/bootstrap4.min.css:2]: About the issue of conflicts, i think it is possible to work on it via the css tags used by bootstrap which are <b>,<strong>but manually adding a fix to these tags will affect other parts of the code base that use the tag. And trying to use classes would mean trying to edit it from the crispy end. i'll work on a quick fix and try to make it stable.

Comment (tristanlatr) [reply saskatoon/sitebase/static/css/bootstrap4.min.css]: So that's a good reason not wanting to add both bootstrap 3 and 4, I think.

**Comment by tristanlatr** [comment]
Hello @nnabuihe-favour, sorry about this but looks like someone has worked on your issue, see #254.
Since I don't want to introduce the whole bootstrap 4 while keeping the bootstrap 3, I'm going to close this PR.

Thanks for your understanding,

**Pull request closed without merging**